compiler/cli/cli-common/src/org/jetbrains/kotlin/cli/common/messages/MessageCollectorExtensions.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.cli.common.messages
+
+fun MessageCollector.reportExtendedNullable(
```

Comment:
This doesn't look nice to me.
@udalov, are you sure that it is better to have such boilerplate, instead of using @JvmName + Suppress?

Reply:
It's not because of it. Let me fix it

Reply:
Removed this extension method.